README.md

```diff
@@ -36,6 +21,19 @@ page.
     </dependency>
```

Comment:
I think that the x.y.z version confuses people. This should be precisely this version so users can cut/paste it.

Reply:
> If this release has no breaking changes relative to 1.1.0 it should be released as 1.1.1, or 1.1.N and not a new minor version.

Breaking changes will be 2.x, 3.x. Non breaking major releases 1.2.x, 1.3.x. Minor, small bug fixing, patch releases will be 1.1.1, 1.1.2 etc.

Reply:
> I think that the x.y.z version confuses people. This should be precisely this version so users can cut/paste it.

Agreed.